pkg/controller/provider/container/ova/collector.go

```diff
@@ -21,7 +21,7 @@ const (
 	// Retry interval.
 	RetryInterval = 5 * time.Second
 	// Refresh interval.
-	RefreshInterval = 10 * time.Second
+	RefreshInterval = 30 * time.Second
```

Comment:
so are we going to update the inventory every 30 seconds?

Reply:
yes, until the watch is fully supported, do you think we need to set a smaller interval?

Reply:
yeah, the collection of OVAs is expected to be relatively stable - I'd change the frequency to execute it every 5 minutes even

Reply:
@bkhizgiy I think you missed this thread

Reply:
I did:) changing